owned-manifests/baremetal_operator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metal3-baremetal-operator
+  namespace: {{ .TargetNamespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: metal3-baremetal-operator
+  template:
+    metadata:
+      labels:
+        name: metal3-baremetal-operator
+    spec:
```

Comment:
Is the intention to also launch ironic here as per https://github.com/metal3-io/baremetal-operator/blob/master/deploy/operator_ironic.yaml

Reply:
Yes. That is correct.